src/DoubleTheDonation/FormExtension/Email/EmailTags.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace GiveDoubleTheDonation\DoubleTheDonation\FormExtension\Email;
+
+use GiveDoubleTheDonation\DoubleTheDonation\Helpers\DoubleTheDonationApi;
+
+/**
+ * @unreleased
+ */
+class EmailTags
+{
+    /**
+     * Register email tags
+     *
+     * @unreleased
+     */
+    public function register(array $tags): array
+    {
+        return array_merge($tags, [
+            [
+                'tag'     => 'matching_company',
+                'desc'    => __(
+                    'Matching Company',
+                    'give-double-the-donation'
+                ),
+                'func'    => [$this, 'handleTag'],
+                'context' => 'dtd',
+            ],
+            [
+                'tag'     => 'matching_company_guidelines',
+                'desc'    => __(
+                    'Matching Company Guidelines',
+                    'give-double-the-donation'
+                ),
+                'func'    => [$this, 'handleTag'],
+                'context' => 'dtd',
```

Comment:
Looks like we never had email tags in v2 forms huh?

I'm wondering if we should prefix the tags and descriptions so the add-on is more clear like  `{double_the_donation_matching_company}`
 _Double the Donation Matching Company_

Do you think that's necessary?

Reply:
We could, but I don't think it's necessary. It is already somewhat prefixed with `matching_`. We could update the tag description instead.

Reply:
Okay, let's go ahead and update the descriptions then to help clarify for customers 👍

Reply:
Resolved https://github.com/impress-org/give-double-the-donation/pull/20/commits/3f180c29e4e224c8242bb94b689fa570e949ed41